resources/field.js

```
@@ -10,7 +11,7 @@ export default {
     reload() {
       if (this.field.reload && queue.allowsReload()) {
         window.setTimeout(() => {
-          this.$router.go();
+          Inertia.reload()
```

This change does not seem to have any effect? How did you test this?

---

You can test it by using `->reload()` function.
It should throw errors about `Cannot read properties of undefined (reading go)`, and it works after change to Inertia reload function